lib/import/mobility.js

```diff
@@ -0,0 +1,192 @@
+var ArgumentParser = require('argparse').ArgumentParser;
+var mongoose = require('mongoose');
+var csv = require('fast-csv');
+var fs = require('fs');
+
+var config = require('../../config');
+var Mobility = require('../../app/models/mobility');
+
+/**
+ * Convert date from EST to ISO format
+ *
+ * @param{string} date - asdf
+ * @return{date}.
+ */
+function prep_isodate(date) {
+  date = new Date(date);
+  date.setHours(date.getHours());
+  return date.toISOString();
+}
+
+/**
+ * Import mobility from CSV to Mongo.
+ *
+ * @param{array} file - file in directory to read and import
+ * @param{string} country_code - lowercase ISO 3166-1 alpha-2 country code
+ * https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2
+ * @param{bool}verbose - whether to log messages while importing.
+ * @return{promise} Fulfilled once all records are saved.
+ */
+function import_file(file, country_code, verbose) {
+  var stream = fs.createReadStream(file);
+  var count = 0;
+  var mobility_objects = [];
+  // TODO(mikefab): consider reject here
+  return new Promise(function(resolve) {
+    var csvStream = csv()
+        .on('data', function(data) {
+          if (verbose && count % 1000 === 0) {console.log(count);}
+          // Skip first line (column names) in each mobility file
+          if ((count > 0) && (data.length > 3)) {
```

Comment:
1. `fast-csv` has options for handling columns, and also error handling for records with too few fields (see the `header` and `strictColumnHandling` options in the docs: https://github.com/C2FO/fast-csv). We should use those features instead of manually keeping a count.

2. Also, I'm pretty sure the `data.length > 3` check doesn't actually work: when I used this code, I got records with missing origin/destination admins.
```
node> > csv.fromString('1,2,3\na,,c\n').on('data', function(line) { console.log(line.length, line); })
3 [ '1', '2', '3' ]
3 [ 'a', '', 'c' ]
```